bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/clangd/ClangdConfigurationFileManager.java

```diff
@@ -238,4 +243,29 @@ private boolean createClangdConfigFile(IFile configFile, String charset, String
 		return false;
 	}
 
+	/**
+	 * Show error dialog to user
+	 * @param title
+	 * @param errorText
+	 * @param status
+	 */
+	private static void showErrorDialog(final String title, final String errorText, final Status status) {
+		UIJob job = new UIJob("Yaml error") //$NON-NLS-1$
+		{
+			@Override
+			public IStatus runInUIThread(IProgressMonitor monitor) {
+				ErrorDialog.openError(getActiveShell(), title, errorText, status);
+				return Status.OK_STATUS;
+			}
+		};
+		job.setSystem(true);
+		job.schedule();
+	}
+
+	private static Shell getActiveShell() {
+		if (PlatformUI.getWorkbench() != null && PlatformUI.getWorkbench().getActiveWorkbenchWindow() != null)
```

Comment:
> `PlatformUI.getWorkbench() != null` 

This code will not work as expected for not initialized workbench
But the issue is different: please avoid static calls from dynamic context, there will be no good. If you hrdly need workbench - please bind it as a service. However, I believe this service does not need UI, see my comment above.